tools/lsp-wake/main.cpp

```diff
@@ -93,7 +93,7 @@ class LSPServer {
   };
 
   MethodResult processRequest(const std::string &requestString) {
-    wcl::log::info(requestString.c_str())({{"rpc", "rx"}});
+    wcl::log::info("%s", requestString.c_str())({{"rpc", "rx"}});
```

Comment:
I think we should understand 1) why the warning attributes we added didn't tell us about this and 2) Why this is even a bug in WASM in the first place as it should (at runtime anyway) be valid.